pydis_core/utils/logging.py

```diff
@@ -9,6 +9,8 @@
     LoggerClass = logging.getLoggerClass()
 
 TRACE_LEVEL = 5
+LOGGING_FORMAT_STRING = "%(asctime)s | %(levelname)s | %(name)s | %(message)s"
+log_format = logging.Formatter(LOGGING_FORMAT_STRING)
```

Comment:
Is this exposed in the documentation? Or is that difficult for stuff in the module scope?

Reply:
Looks good, thanks!